drivers/hmis/app/models/hmis/hud/processors/employment_education_processor.rb

```diff
@@ -0,0 +1,20 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Hmis::Hud::Processors
+  class EmploymentEducationProcessor < Base
+    def factory_name
+      :employment_education_factory
+    end
+
+    def schema
+      Types::HmisSchema::EmploymentEducation
+    end
+
+    def information_date(_)
+    end
```

Comment:
EmploymentEducation does have an information date, so I think this should be removed to keep the default behavior for setting that
```suggestion
```

Reply:
Yeah this needed to be removed so that the information date gets updated when the assessment date changes. I implemented while testing since I wasn't sure